SpatialGDK/Source/SpatialGDK/Public/Utils/OpCallbackTemplate.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include <WorkerSDK/improbable/c_worker.h>
+#include "OpCallbackTemplate.generated.h"
+
+UCLASS(abstract)
+class SPATIALGDK_API UOpCallbackTemplate : public UObject
+{
+	GENERATED_BODY()
+
+public:
+	UOpCallbackTemplate() = default;
+	~UOpCallbackTemplate() = default;
+
+	/**
+	  * Called by SpatialDispatcher on initialization, this is to allow the user to write more useful callbacks.
+	  * @param World the UWorld top level hierarchy object.
+	  */
+	void Init(UWorld* InWorld) {
```

Comment:
```suggestion
	void Init(UWorld* InWorld)
	{
```

Reply:
Actually, can you use an initializer list instead?

Reply:
Is it possible to pass constructor arguments when calling `NewObject`?

Reply:
Unfortunately not :(

Reply:
Was there a delete comment here or am I imagining things?

Reply:
Yes, sorry, deleted a few seconds after posting when I saw this wasn't the constructor, but imagine you got an email or similar. Bad form on me.

Reply:
Now I look CRAZY 😱😱😱😱😱(np 😄)